### PR TITLE
Add model tests

### PR DESCRIPTION
**Pull request by konifar**
## Overview
- Refactored the logic to check the current locale.
- Added some simple model tests.

Thanks for giving the great app and conference! 🎉

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -22,6 +22,7 @@ dev_dependencies:
   flutter_launcher_icons: "^0.6.0"
   build_runner: ^0.9.0
   json_serializable: ^0.5.4
+  mockito: ^3.0.0
```
Comment (konifar) [inline pubspec.yaml:25]: Actually mockito is not used in this PR but I'm sure this helpful to write tests.

**Comment by konifar** [comment]
Oh, CircleCI test always fail because of No Tests matter 😢 
https://circleci.com/gh/mercari/mtc2018-app

I don't know why both Travis and CircleCI are needed but I'll add `.circleci/config.yml` in this PR!

**Comment by konifar** [comment]
Added. I hope all the checks go well.
https://github.com/mercari/mtc2018-app/pull/64/commits/f7b97bef9c69dc1788f69a17f3c04cdf0da48da1

**Review by masamichiueta — APPROVED** [review]
LGTM🎉
Thank you for your great refactoring, adding tests, and contribution to MTC🙏

**Pull request merged**